_commands/basics/date.md

```diff
@@ -33,28 +34,35 @@ The useful options include:
 %s  show the current seconds from 1970-01-01 00:00:00 UTC
 ~~~
 
-For more help, run
+Here are some examples of using options:
+
 ~~~ bash
-man date
+$ date +%a%b
+TueFeb
 ~~~
-to view the manual of this command.
-
-#### Example command
 
 ~~~ bash
 $ date +%s
 1517939576
 ~~~
 
-##### Break it down
+The syntax for using flags is:
```

Comment:
Could you provide examples and descriptions of a few other flags? For example, what does the `-r`, `-f` and `-s` flags do?